contrib/vcl/source/BUILD

```diff
@@ -28,37 +32,57 @@ cc_library(
     visibility = ["//visibility:public"],
 )
 
-genrule(
+envoy_cmake(
     name = "build",
-    srcs = [
-        "@com_github_fdio_vpp_vcl//:all",
+    lib_source = "@com_github_fdio_vpp_vcl//:all",
+    out_static_libs = [
+        "libvppcom.a",
+        "libvppinfra.a",
+        "libsvm.a",
+        "libvlibmemoryclient.a",
     ],
+    cache_entries = {
+        "CMAKE_BUILD_TYPE": "Release",
+        "VPP_API_TEST_BUILTIN": "OFF",
+        "BUILD_SHARED_LIBS": "OFF",
+        "CMAKE_ENABLE_TESTING": "OFF",
+        "CMAKE_ENABLE_EXPORTS": "OFF",
+    },
+    default_cache_entries = {},
+    # build_args = ["--", "-C", ".", "vppcom"],
+    copts = ["-Wno-unused-variable"],
+    linkopts = ["-Wno-unused-variable"],
+    working_directory = "src",
+    postfix_script = """
+        mkdir -p $INSTALLDIR/lib/external $INSTALLDIR/include/external \
+        && find . -name "*.a" | xargs -I{} cp -a {} $INSTALLDIR/lib/ \
+        && find . -name "*.h" | xargs -I{} cp -a {} $INSTALLDIR/include
+    """,
+    tags = ["skip_on_windows"],
+    # tags = ["no-remote", "skip_on_windows"],
+    targets = ["", "install"],
+    build_data = [requirement("ply")],
+    env = {
+        "PLYPATHS": "$(locations %s)" % requirement("ply"),
```

Comment:
there is probably a more elegant way to do this - but i was struggling to think of one